### PR TITLE
Time Advancement

### PR DESCRIPTION
**Pull request by naam1595am**

### CODE REVIEW

File: app/models/player.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 
-# player character model
+# Player character model
 class Player < Character
   validates :name, :occupation, :inventory_slots, :balance, presence: true
   validates :current_level, presence: true, numericality: { greater_than: 0 }
+  validates :day, numericality: { greater_than_or_equal_to: 1, only_integer: true }, presence: true
+  validates :hour, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 11, only_integer: true },
+                   presence: true
+  validates :era, numericality: { greater_than_or_equal_to: 1, only_integer: true }, presence: true
```
Comment (KiranWells) [inline app/models/player.rb]: The era should be tracked by the level; we don't need a separate variable for it

**Review by KiranWells — CHANGES_REQUESTED** [review]
The era can be removed since that is a duplicate of the current_level field, but aside from that this looks good

File: app/models/player.rb (hunk 1 above)
Comment (naam1595am) [reply app/models/player.rb]: Completed.

**Review by KiranWells — APPROVED** [review]
looks good

**Pull request merged**